ipaserver/install/kra.py

```diff
@@ -47,9 +49,9 @@ def install_check(api, replica_config, options):
 
     if replica_config is not None:
         if not api.Command.kra_is_enabled()['result']:
-            raise RuntimeError(
-                "KRA is not installed on the master system. Please use "
-                "'ipa-kra-install' command to install the first instance.")
+            root_logger.info("KRA is not installed on the master system. "
```

Comment:
Hm, there's a point to that. I propose moving this message to `KRAInstance.configure_instance` but as you say, it should be a debug message instead.